wcfsetup/install/files/lib/system/email/Mailbox.class.php

```diff
@@ -26,6 +26,12 @@ class Mailbox {
 	 */
 	protected $name = null;
 	
+	/**
+	 * The preferred language for this mailbox.
```

Comment:
```suggestion
	 * The preferred language of this mailbox.
```

Reply:
I would have applied the suggested change myself, but unfortunately you did not tick the checkbox to allow the maintainers of WoltLab/WCF to push in your branch.